edflow/data/believers/meta_loaders.py

```diff
@@ -64,6 +75,39 @@ def numpy_loader(path):
     """
 
     def loader():
-        return np.load(path)
+        path_ = os.path.join(root, path)
+        return np.load(path_)
+
+    loader.__doc__ = f"""Loads the numpy array found at {path} relative to
+    {root}."""
 
     return loader
+
+
+def category(index, categories):
+    """
+
+    Parameters
+    ----------
+    index : int, Hashable
+        Some value that will be passed to :attr:`categories`'s
+        :meth:`__getitem__` method. I.e. :attr:`categories` can be a ``list`` or
+        ``dict`` or whatever you want!
+    categories : list, dict, object with `__getitem__` method
+        Defines the categories you have in you dataset. Will be accessed like
+        ``categories[index]``
+
+    Returns:
+        category : object
+            ``categories[index]``
+    """
+
+    return categories[index]
+
+
+DEFAULT_LOADERS = {
+    "image": image_loader,
+    "np": numpy_loader,
+    "category": category,
+    "cat": category,
```

Comment:
two names for category?

Reply:
Yes. This allows to specify the loader in the `meta.yaml` using its full name and keeping things human readable, while using the short name in the name of the label array if one wants to use this variant of the default loader syntax.

First case:
```
# meta.yaml
loaders:
    pid: category
loader_kwargs:
    pid:
        category: ['besser', 'pesser']
```

Second case:
Name of the label file
```
<meta_dir>/labels/pid:cat-*-<shape>-*-<type>.npy
```
```
# meta.yaml
loader_kwargs:
    pid:
        category: ['besser', 'pesser']
```

Both cases will yield equivalent results.